lib/src/core/themes/custom_themes/elevated_button_theme.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_clean_architecture/src/core/styles/app_colors.dart';
+
+class TElevatedButtonTheme {
+  TElevatedButtonTheme._(); // to avoid creating instances
+
+  /* Light Theme */
+  static final ElevatedButtonThemeData lightElevatedButtonTheme =
```

Comment:
please move these to styles

and have base style and then create light and dark theme from base theme